src/core/editform/qgsattributeeditorelement.cpp

```diff
@@ -22,14 +22,17 @@
 #include "qgsattributeeditorhtmlelement.h"
 #include "qgsattributeeditorqmlelement.h"
 #include "qgsattributeeditorrelation.h"
-
+#include "qgssymbollayerutils.h"
 
 
 QDomElement QgsAttributeEditorElement::toDomElement( QDomDocument &doc ) const
 {
   QDomElement elem = doc.createElement( typeIdentifier() );
   elem.setAttribute( QStringLiteral( "name" ), mName );
   elem.setAttribute( QStringLiteral( "showLabel" ), mShowLabel );
+  elem.setAttribute( QStringLiteral( "labelColor" ), QgsSymbolLayerUtils::encodeColor( mLabelColor ) );
+  elem.setAttribute( QStringLiteral( "labelFont" ), mLabelFont.toString() );
```

Comment:
There's methods in QgsFontUtils to encode a font -- special logic is required to do this.